otcextensions/sdk/vlb/v3/load_balancer.py

```diff
@@ -29,20 +29,37 @@ class LoadBalancer(resource.Resource):
         'description', 'flavor_id', 'name', 'project_id', 'provider',
         'vip_address', 'vip_network_id', 'vip_port_id', 'vip_subnet_id',
         'vip_qos_policy_id', 'provisioning_status', 'operating_status',
-        'availability_zone', is_admin_state_up='admin_state_up',
+        'availability_zone_list', is_admin_state_up='admin_state_up',
+        availability_zone='availability_zone_list',
     )
 
     # Properties
-    #: The administrative state of the load balancer *Type: bool*
-    is_admin_state_up = resource.Body('admin_state_up', type=bool)
     #: Name of the target Octavia availability zone
-    availability_zone = resource.Body('availability_zone', type=list)
+    availability_zone = resource.Body('availability_zone_list', type=list)
+    #: Billing information about the load balancer.
+    billing_info = resource.Body('billing_info')
     #: Timestamp when the load balancer was created
     created_at = resource.Body('created_at')
     #: The load balancer description
     description = resource.Body('description')
+    #: EIP bound to the load balancer.
+    eips = resource.Body('eips', type=list, list_type=dict)
     #: The load balancer flavor ID
     flavor_id = resource.Body('flavor_id')
+    #: Specifies whether the load balancer is a dedicated load balancer.
+    guaranteed = resource.Body('guaranteed')
+    #: Load balancer ID.
+    id = resource.Body('id')
```

Comment:
id/name are general attrs in the base class and they do not need to be explicitly included